externals/paho-mqtt/v0/src/index.kt

```diff
@@ -0,0 +1,63 @@
+//headers for Paho JavaScript Client
+//The Paho JavaScript Client is an MQTT browser-based client library written in Javascript that uses WebSockets to connect to an MQTT Broker.
+//https://github.com/eclipse/paho.mqtt.javascript
+
+package js.externals.paho_mqtt
+
+external object Paho {
+    object MQTT {
+        class Client(hostname: String, port: Int, clientId: String) {
+            val hostname: String
```

Comment:
According [the docs](http://www.eclipse.org/paho/files/jsdoc/Paho.MQTT.Client.html) it should be called `host`.